tests/chains/ethereum/scripts/cmd/helper/chain.go

```diff
@@ -0,0 +1,266 @@
+package helper
+
+import (
+	"context"
+	"crypto/ecdsa"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"log"
+	"math/big"
+	"os"
+
+	"github.com/ethereum/go-ethereum/accounts/abi/bind"
+	"github.com/ethereum/go-ethereum/common"
+	gethtypes "github.com/ethereum/go-ethereum/core/types"
+	gethcrypto "github.com/ethereum/go-ethereum/crypto"
+	"github.com/hyperledger-labs/yui-ibc-solidity/pkg/client"
+	ibccommitment "github.com/hyperledger-labs/yui-ibc-solidity/pkg/contract/ibccommitmenttesthelper"
+	"github.com/hyperledger-labs/yui-ibc-solidity/pkg/contract/ibchandler"
+	"github.com/hyperledger-labs/yui-ibc-solidity/pkg/contract/ics20bank"
+	"github.com/hyperledger-labs/yui-ibc-solidity/pkg/contract/ics20transferbank"
+	"github.com/hyperledger-labs/yui-ibc-solidity/pkg/contract/simpletoken"
+	"github.com/hyperledger-labs/yui-ibc-solidity/pkg/wallet"
+)
+
+type PathConfig struct {
+	Src      PathInfo `json:"src"`
+	Dst      PathInfo `json:"dst"`
+	Mnemonic string   `json:"mnemonic"`
+}
+
+type PathInfo struct {
+	ChainID      string `json:"chain-id"`
+	ClientID     string `json:"client-id"`
+	ConnectionID string `json:"connection-id"`
+	ChannelID    string `json:"channel-id"`
+	PortID       string `json:"port-id"`
+	Order        string `json:"order"`
+	Version      string `json:"version"`
+}
+
+type ChainConfig struct {
+	Chain  ChainInfo    `json:"chain"`
+	Prover ProverConfig `json:"prover"`
+}
+
+type ChainInfo struct {
+	Type        string `json:"@type"`
+	ChainID     string `json:"chain_id"`
+	EthChainID  int    `json:"eth_chain_id"`
+	RPCAddr     string `json:"rpc_addr"`
+	HDWMnemonic string `json:"hdw_mnemonic"`
+	HDWPath     string `json:"hdw_path"`
+	IBCAddress  string `json:"ibc_address"`
```

Comment:
Maybe, you can reuse `yui-ibc-solidity/relay/ethereum.ChainConfig` here?

Reply:
https://github.com/datachainlab/yui-relayer-build/pull/5/commits/a9e763df1bd7e5f26d50b1087bc06ab23d560b95
Changed to use reusable structs